src/socket.rs

```diff
@@ -1017,6 +1119,28 @@ impl Drop for Socket {
     }
 }
 
+/// Fix-up a created socket by setting the proper portability flags.
+#[allow(unused)]
+fn fixup_socket(mut sock: Socket) -> io::Result<Socket> {
```

Comment:
Better name would be `set_common_flags` or something, but `fixup` shouldn't be in it.

Reply:
renamed